src/main/java/uk/gov/companieshouse/api/accounts/model/entity/notes/CreditorsWithinOneYearEntity.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.companieshouse.api.accounts.model.entity.notes;
+
+import com.google.gson.Gson;
+import org.springframework.data.mongodb.core.mapping.Document;
+import uk.gov.companieshouse.api.accounts.model.entity.BaseEntity;
+
+import java.util.Objects;
+
+@Document(collection = "notes")
+public class CreditorsWithinOneYearEntity extends BaseEntity {
+
+    CreditorsWithinOneYearDataEntity data;
+
+    public CreditorsWithinOneYearDataEntity getData() {
+        return data;
+    }
+
+    public void setData(CreditorsWithinOneYearDataEntity data) {
+        this.data = data;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Curly braces required by our coding standards even when they are syntactically not needed.

Reply:
done